src/SidePanel/sidepanel.scss

```diff
@@ -4,8 +4,66 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
  */
 
+@import '~pc-nrfconnect-shared/styles';
+
 .sidepanel {
     .form-label {
         margin-bottom: 0;
     }
+
+    .h-24 {
+        height: 24px;
+    }
+
+    .active-anim {
+        overflow: hidden;
+        position: relative;
+
+        &::after {
+            content: '';
+            background: $gray-700;
+            height: 2px;
+            position: absolute;
+            bottom: 0;
+            left: 0;
+            right: 0;
+            animation: snake 4s cubic-bezier(0.5, 0, 0.5, 1) infinite;
+        }
+
+        @keyframes snake {
+            0% {
+                transform: translate(-100%);
+            }
+            100% {
+                transform: translate(100%);
+            }
+        }
+    }
+
+    .reset-btn {
+        width: 100%;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        margin-top: 8px;
+        padding: 0;
+        height: 24px;
+        border-color: $gray-700;
+    }
+    
+    .start-stop {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        margin-top: 24px;
+        padding: 0;
+        height: 34px;
+        border-color: $gray-700;
+
+        span {
+            color: $gray-700;
+            font-size: 20px;
+            margin-right: 8px;
+        }
+    }
 }
```

Comment:
Will try to move this to shared later, so that the better buttons could easily replace the old ones in other apps without having to implement this in every app.